src/internal/minikubetestenv/deploy.go

```diff
@@ -22,36 +25,69 @@ import (
 )
 
 const (
-	helmChartLocalPath     = "../../../etc/helm/pachyderm"
 	helmChartPublishedPath = "pach/pachyderm"
-	helmRelease            = "test-release"
-	ns                     = "default"
 	localImage             = "local"
 	licenseKeySecretName   = "enterprise-license-key-secret"
 )
 
-func getPachAddress(t testing.TB) *grpcutil.PachdAddress {
-	cfg, err := config.Read(false, false)
+type DeployOpts struct {
+	Version      string
+	Enterprise   bool
+	AuthUser     string
+	CleanupAfter bool
+	// Because NodePorts are cluster-wide, we use a PortOffset to
+	// assign separate ports per deployment.
+	// NOTE: it might make more sense to declare port instead of offset
```

Comment:
We might want to have a data structure which maps namespace name to ports. I'm encountering the same issue on the pulumi side setting up multiple pachs in a given cluster.